lib/spaceship/tunes/tunes_client.rb

```diff
@@ -342,6 +342,70 @@ def update_app_version!(app_id, version_id, data)
     end
 
     #####################################################
+    # @!group AppAnalytics
+    #####################################################
+    def time_last_7_days
+      time = Time.now
+      past = time - (60 * 60 * 24 * 7)
+      end_t   = time.strftime("%Y-%m-%dT00:00:00Z")
+      start_t = past.strftime("%Y-%m-%dT00:00:00Z")
+
+      return start_t, end_t
+    end
+
+    def time_last_30_days
+      time = Time.now
+      past = time - (60 * 60 * 24 * 30)
+      end_t   = time.strftime("%Y-%m-%dT00:00:00Z")
+      start_t = past.strftime("%Y-%m-%dT00:00:00Z")
+
+      return start_t, end_t
+    end
+
+    def time_last_90_days
+      time = Time.now
```

Comment:
Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.